cmd/xgen/xgen.go

```diff
@@ -62,13 +63,14 @@ var SupportLang = map[string]bool{
 func parseFlags() *Config {
 	iPtr := flag.String("i", "", "Input file path or directory for the XML schema definition")
 	oPtr := flag.String("o", "xgen_out", "Output file path or directory for the generated code")
+	noXmlName := flag.Bool("noxml", false, "Don't add xmlName fields")
```

Comment:
I suggest that using the `--skip-go-xml-name` instead of `noxml`, will be more clear.